.github/workflows/build.yml

```diff
@@ -132,14 +129,69 @@ jobs:
         name: reactos-syms-msvc${{matrix.toolset}}-${{matrix.arch}}-${{matrix.config}}-${{github.sha}}
         path: build/msvc_pdb
 
+  build-msvc_win2019:
+    runs-on: windows-2019
+    strategy:
+      matrix:
+        toolset: ['14.0'] # VS 2015
+        # vsversion: ['2019'] # Use VS2019 with older toolset
+        arch: [i386, amd64]
```

Comment:
amd64 jobs still fail: why add it, yet?

Reply:
It seems amd64 build has been broken in 93d8a1b7b.

Reply:
I wanted to test whether amd64 works, and it generally works except somewhere for the generation of the 16-bit asm files....

Reply:
it's now fixed.